gpii/node_modules/processReporter/src/processesBridge.js

```diff
@@ -137,21 +144,37 @@ gpii.processes.findProcessByPid = function (that, pid, procArray) {
     }, null);
 };
 
+/**
+ * Return the path of a running process.
+ *
+ * @param {Component} that - an instance of a processes component.
+ * @param {Number} pid - the process id of the process.
+ * @return {String} - The full path of the process, or null if there's no matching process.
+ */
+gpii.processes.getProcessPath = function (that, pid) {
```

Comment:
How did this material end up in this pull? Is it contamination from another pull?

Reply:
Yes,  https://github.com/GPII/windows/pull/170 (and this one https://github.com/GPII/windows/pull/169).
It's a dependent.

Reply:
But there must be a universal pull that it came from?

Reply:
ah! wrong repo.

found it: https://github.com/GPII/universal/pull/600